README.md

```diff
@@ -377,13 +377,10 @@ The -s makes it so that any print statements aren't swallowed if tests pass. The
 Troubleshooting
 --------------
 
-### ImportErrors
+### Deprecated
 
-If you get import errors, make sure the directory `./vendor/src/` is not empty
-or contains only directories that are empty. If they are empty, run the
-following command:
-
-``git submodule update --init --recursive``
+Make sure you don't have a ./vendor or ./vendor-local directory. This functionality has been deprecated. All dependencies
```

Comment:
Also, can you make the lines no longer than 80 characters each. If you look throughout the rest of the file, you'll notice most lines are capped before 80 characters